src/install.rst

```diff
@@ -6,148 +6,184 @@ Installing Nextstrain
+         .. code-block:: bash
 
-      Whenever you open a new terminal window to work on a Nextstrain analysis, remember to activate the Nextstrain Conda environment with ``conda activate nextstrain``.
-      Next, try :doc:`tutorials/quickstart`.
+            nextstrain check-setup --set-default
 
-   .. group-tab:: Docker
+         The final output from the last command should look like this, where ``<option>`` is the environment chosen in the previous step:
 
-    `Docker <https://docker.com/>`_ is a container system freely-available for all platforms.
-    When you use the Nextstrain CLI with Docker, you don’t need to manage any other Nextstrain software dependencies as validated versions are already bundled into `a container image by the Nextstrain team <https://github.com/nextstrain/docker-base/>`_.
+         .. code-block:: none
 
-    First, `follow Docker's installation guide <https://docs.docker.com/engine/install/>`_ for your operating system.
-    After installing and starting Docker, create a Conda environment named ``nextstrain``.
-    This command will install the Nextstrain CLI and Git (a dependency of subsequent tutorials).
+            Setting default environment to <option>.
 
-    .. code-block:: bash
+   .. group-tab:: Windows
 
-        conda create -n nextstrain -c conda-forge -c bioconda nextstrain-cli git
+      .. note::
 
-    Confirm that the installation worked and configure the CLI to use Docker as the default environment manager.
+         Due to installation constraints, there is no way to use the native build/view environment on Windows directly. Follow steps for **WSL on Windows** if the native environment is desired.
 
-    .. code-block:: bash
+      1. Install `Anaconda or Miniconda <https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html>`_.
+         Miniconda is sufficient for this guide.
+      2. Install Visual C++ build tools following `this guide <https://stackoverflow.com/a/64262038>`_.
 
-        conda activate nextstrain
-        nextstrain check-setup --set-default
+         - This is necessary for a dependency of Nextstrain CLI and `we are investigating options to eliminate this step <https://github.com/nextstrain/cli/issues/31#issuecomment-970641263>`_.
 
-    The final output from the last command should look like this:
+      3. Open an Anaconda Prompt, which can be found in the Start menu.
+      4. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
 
-    .. code-block:: bash
+         .. code-block:: none
 
-        Setting default environment to docker.
+            conda create -n nextstrain -c bioconda nextstrain-cli --yes
+            conda activate nextstrain
 
-    Finally, download the latest Docker image for Nextstrain.
+      5. Install the Nextstrain build/view environment by `installing Docker Desktop with WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
+      6. Confirm that the installation worked.
```

Comment:
I'm curious about your experience with this, @victorlin, but in my efforts to install Docker Desktop and WSL 2, I had to restart my computer at least once (after installing the Linux kernel update prerequisite for Docker). Was that the case for you, too?

If so, we should warn the reader that they will need to restart their computer during Nextstrain installation (so it doesn't come as an annoying surprise halfway through installing our dependencies) and also consider adding a reminder to open a new Anaconda prompt before this "Confirm that the installation worked" step.

It's actually worth clearly stating for users in any case that they need to run Nextstrain from an Anaconda prompt.

Reply:
The Docker and WSL installs were system-wide so I wasn't able to check this on a new user account (and I don't feel like wiping and re-installing Windows yet again). But I vaguely remember restarting at least once for WSL, so I've added that as a note: https://github.com/nextstrain/docs.nextstrain.org/pull/81/commits/0a7153e155c5f77383ce1f90878c1f1c444726aa